javascript/javascript1/week2/homework/main-4.js

```diff
@@ -0,0 +1,39 @@
+//STUDENT MANAGER
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if (studentName === "")  { //excludes empty string
+        return ("Please, add a student name!");
+    }
+
```

Comment:
Just a styling issue again, but try not to leave an empty line between `if` and `else`. The `else` part is basically a continuation of the `if` so shouldn't be separated. Again this is something Prettier will fix for you.